### ISSUE TITLE
[Feature request] Revert Facebook profile links to human-readable usernames, rather than UUID

### ISSUE THREAD

**Issue by Darthagnon** [zagushka/birthday-calendar-extractor#24]
Following up from our email conversation: Any chance at some point you could please revert the exported  Facebook profile links to the human-readable ```https://www.facebook.com/facebook.username```, rather than using UUIDs ```https://www.facebook.com/fbUUID123456```? Or as a toggle? (as far as I can tell, the main benefit of UUID is saving a few KB in filesize of the export?)

## Use case
Friend has a name like ```Jôșèpħ Sä®MăLé``` with accented letters (probably Asian, Cyrillic and Semitic characters, too). I've seen them mangled in the export. Say this person deletes/disables their FB account. Now all the user has to guess who's birthday is appearing in the ICS/CSV is a mangled name and a dead link with a UUID. Solvable if FB username is used instead of UUID.

I believe you indicated v2.5.2 is the last version that exports FB usernames, instead of UUIDs?

[v2.5.2 binaries on Google Drive that you sent me, in case anyone else has the same problem](https://drive.google.com/file/d/1JTnBHh7TJROqOL5EDUQN-Nc0jocsv-LM/view?usp=sharing)

Thank you so much for considering my feedback! And thank you for making this essential extension! ❤

**Comment by linkwoman** [comment]
I agree. I added my name to the list and realized that my link is the only one with my actual name in the link; all others are UUIDs. Would prefer to see names in the links.
